sys/src/libmach/6.c

```diff
@@ -10,10 +10,10 @@
 /*
  * amd64 definition
  */
-#include <u.h>
+#include <amd64/include/u.h>
```

Comment:
it would be better if libmach json could have a cflag -I/$ARCH/include. Not only for eye candy, better for portability. No?